### PR TITLE
Expire users with Okapi timer (1 minute) fixes MODUSERS-244

### PR DESCRIPTION
**Pull request by adamdickmeiss**
This has been tested on a Vagrant box .. where existing mod-users upgraded to this PR code. There's no behavioral change, except that I thought 1 minute would better than 30 seconds.. Can of course be adjusted.

### CODE REVIEW

File: ramls/users.raml [hunk 1]
```diff
@@ -51,3 +51,17 @@ resourceTypes:
...
+  /expire/timer:
+    post:
+      description: Expire timer (timer event)
+      responses:
+        204:
+          description: Expired ok
+        400:
+          description: User Error
+          body:
+            text/plain:
```
Comment (hjiebsco) [inline ramls/users.raml:63]: Is this needed?  There is no implementation to return 400.

**Review by hjiebsco — APPROVED** [review]
Nice change! Glad to see `select nspname from pg_catalog.pg_namespace where nspname ~ '^[^_]+_mod_users$';` is gone.

File: ramls/users.raml (hunk 1 above)
Comment (adamdickmeiss) [reply ramls/users.raml:63]: RMB may return it - before our handler is called.

**Comment by adamdickmeiss** [comment]
> Nice change! Glad to see `select nspname from pg_catalog.pg_namespace where nspname ~ '^[^_]+_mod_users$';` is gone.

Yes... This code is old, added May 31 2018 , so it first appeared in release mod-users 15.2.0.

Okapi started to offer timer in May 2019.

**Pull request merged**